src/public/partials/debug.eta

```diff
@@ -0,0 +1,75 @@
+<div class="container-sm">
+  <button class="btn btn-info" type="button" data-bs-toggle="collapse" data-bs-target="#debug-panel" aria-expanded="true" aria-controls="debug-panel">Toggle debug panel</button>
```

Comment:
The visual appearance and placement of the debug panel is not ideal. I'm very much open to suggestions on how to improve this.